unlock-protocol-com/blog/crosschain-connext-gnosis.md

```diff
@@ -0,0 +1,79 @@
+
+### Does this implementation enhance security or efficiency in cross-chain DAO governance?
+
+This new implementation helps the Unlock Protocol ecosystem execute more efficiently, since approved proposals will propagate more quickly across the various networks with reduced overhead. 
+
+Security, as usual, is a double-edged sword. On one side, a single proposal can apply changes across multiple chains making audit and tracking of calls easier, and the protocol more consistent (for instance in case of an upgrade, which in the past could result in different protocol versions across different chains). 
+
+Of course, the converse is also true — a potential error will propagate more quickly to the other chains. This upgrade includes multiple safeguards, including a cooldown process for transactions when they arrive on destination chains with the ability to cancel a malicious transaction during that cooldown period. A timelock allows quarantining a call after it has crossed the bridge. 
+
+The main concern is to have a mechanism to prevent the execution of a malicious call in case the bridge itself is ever compromised.
+
+### How does this implementation contribute to the broader goal of decentralization in web3?
+
+Unlock Protocol's DAO (with a D as in “Decentralized”) aims at building tools for the community of Unlock Protocol developers, users and stakeholders to manage the protocol themselves, without intervention from the Unlock Labs core team. 
+
+This cross-chain governance process puts management of contracts previously only managed by Unlock’s team multisig in the hands of the Unlock DAO. By doing so, Unlock also stands at the forefront of experiments in protocol governance by proposing a complete, auditable, and transparent way to make changes in the protocol when necessary.
+
+### What was the role of Connext in this architecture?
+
+Connext provides the bridge technology that is necessary for taking the calls contained in the DAO proposal from mainnet to the other chains. The Connext bridge effectively conveys the instructions to execute across chains.
+
+### What was the role of Safe in this architecture?
+
+Safe provides the receiving end of DAO proposals on destination chains. When a call is sent from the DAO’s Governor contract, it crosses the bridge towards a Safe that receives it.
+
+Safe’s Zodiac suite of modules allows for a Safe to 1) receive calls properly from the bridge and 2) put a call in quarantine for a cooldown period once it is received.
+
+Unlock Protocol has historically relied on Gnosis Safe for the management of the protocol contracts. By reusing and building on the same tech, the changes required for the protocol governance are kept to a minimum to avoid potential security issues.
```

Comment:
```suggestion
Unlock Protocol has historically relied on Safe for the management of the protocol contracts. By reusing and building on the same tech, the changes required for the protocol governance are kept to a minimum to avoid potential security issues.
```